### PR TITLE
hu_HU.properties

### PR DESCRIPTION
**Pull request by TheAdamYT**

### CODE REVIEW

**Comment by RaphiMC** [comment]
Please rename the file to hu_HU.properties

File: src/main/resources/assets/language/hu.properties [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+tab.general.server_address.tooltip=Támogatott Cím:\n- address\n- address:port\n- ClassiCube Direct URL cím
+tab.general.server_version.label=Szerver Verzió:
+tab.general.minecraft_account.label=Minecraft Fiók:
+tab.general.minecraft_account.option_no_account=Ne használj semmilyen fiókot
+tab.general.minecraft_account.option_select_account=Kiválasztott fiók használata
+tab.general.minecraft_account.option_openauthmod=OpenAuthMod Használata
+tab.general.betacraft_auth.label=BetaCraft Auth(Classic)
+tab.general.betacraft_auth.tooltip=BetaCraft bekapcsolása. Fel enged Classic minecraft szerverekre.
+tab.general.state.loading=ViaProxy Betöltése...
+tab.general.state.start=Indít
```
Comment (FlorianMichael) [inline src/main/resources/assets/language/hu.properties:30]: deepl.com translates this to "Indicate" which is wrong, it should mean Start

File: src/main/resources/assets/language/hu.properties [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+tab.general.state.starting=Indítás...
```
Comment (FlorianMichael) [inline src/main/resources/assets/language/hu.properties:31]: same here

File: src/main/resources/assets/language/hu.properties [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+tab.realms.unsupported_account=Nem támogatott fiók kiválasztva
+tab.realms.refreshing_account=Fiók frissítése...
+tab.realms.availability_check=Elérhetőség ellenőrzése...
+tab.realms.unavailable=A ViaProxy-od elavult és nem kompatibilis a Minecraft Realms-szal.\nKérlek, használd a ViaProxy legújabb verzióját!.
+tab.realms.loading_worlds=Világok betöltése...
+tab.realms.no_worlds=Nem találtunk világokat
+tab.realms.account=Fiók: %s
+tab.realms.release=Kiadás
+tab.realms.snapshot=Snapshot
+tab.realms.join=Fellépés
```
Comment (FlorianMichael) [inline src/main/resources/assets/language/hu.properties:96]: This should mean join, not leave

File: src/main/resources/assets/language/hu.properties [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+tab.realms.joining=Fellépés...
```
Comment (FlorianMichael) [inline src/main/resources/assets/language/hu.properties:97]: same here

File: src/main/resources/assets/language/hu.properties (hunk 1 above)
Comment (TheAdamYT) [reply src/main/resources/assets/language/hu.properties:30]: i only used deepl for some things that i didn't know

File: src/main/resources/assets/language/hu.properties (hunk 3 above)
Comment (TheAdamYT) [reply src/main/resources/assets/language/hu.properties:96]: join = fellépés ???

File: src/main/resources/assets/language/hu.properties (hunk 4 above)
Comment (TheAdamYT) [reply src/main/resources/assets/language/hu.properties:97]: Lelépés means leave fellépés means join

File: src/main/resources/assets/language/hu.properties (hunk 3 above)
Comment (FlorianMichael) [reply src/main/resources/assets/language/hu.properties:96]: Wrong line, sorry about that

**Comment by TheAdamYT** [comment]
> Please rename the file to hu_HU.properties

i renamed it

**Pull request closed without merging**